Pod/Classes/VOKCoreDataManager.m

```diff
@@ -594,18 +619,22 @@ - (NSURL *)applicationLibraryDirectory
 
 - (void)resetCoreData
 {
-    NSArray *stores = [[self persistentStoreCoordinator] persistentStores];
+    if (_persistentStoreCoordinator) {
```

Comment:
Because the method actually tries to spin up the whole stack again if nothing's there, and if you were testing, say, a bundle which doesn't have a managed object model to make sure it throws, then when it gets spun up again here then it will throw again.